src/components/ProjectCard/ProjectCard.js

```diff
@@ -61,13 +61,19 @@ const ProjectCard = ({ project, handleModal, handleKeyPress, handleProjectDelete
     </section>
   })
   return (
-    <article className='project-card'>
-      <img className='img--close' src={close} onClick={() => handleProjectDelete(project.id)} />
-      <h2 contenteditable="true" onKeyPress={(e) => handleKeyPress(e, project.id, e.target.innerText)}>{project.name}</h2>
+    <article className="project-card">
+      <img 
+        className="img--close" 
+        src={close}
+        alt="x to close button" 
+        onClick={() => handleProjectDelete(project.id)} />
+      <h2 
+        className="project-name"
+        contenteditable="true" 
+        onKeyPress={(e) => handleKeyPress(e, project.id, e.target.innerText)}>{project.name}</h2>
```

Comment:
I like the way you broke this img and h2 out. Do you think we should break out the {project.name} onto its own line and indent it in?